commands/pipelines/promote.js

```diff
@@ -156,8 +197,9 @@ module.exports = {
       heroku, promotionActionName, coupling.pipeline.id, coupling.app.id, targetApps
     )
 
-    const pollLoop = pollPromotionStatus(heroku, promotion.id)
-    const promotionTargets = yield cli.action('Waiting for promotion to complete', pollLoop)
+    const pollLoop = pollPromotionStatus(heroku, promotion.id, true)
+    var promotionTargets = yield cli.action('Waiting for promotion to complete', pollLoop)
```

Comment:
This is so minor that I almost wouldn't bother commenting but could this be a `let` instead of `var` please?